config/routes.rb

```diff
@@ -6,7 +6,7 @@
   resources :users
   resources :sessions
 
-  post "posts/:id/edit" => "posts#update"
+  # post "posts/:id/edit" => "posts#update"
```

Comment:
Better to just remove this than leave it as a comment. If we need to refer back to it we will have it our commit history anyway.